services/paymaster/src/constraint.rs

```diff
@@ -293,37 +285,33 @@ impl ContextualPubkey {
                 }
             }
 
-            ContextualPubkey::Signer { index } => {
-                let index_uint = if *index >= 0 {
-                    *index as usize
-                } else if (-*index as usize) <= signers.len() {
-                    signers.len() - (-*index as usize)
+            ContextualPubkey::FeePayer => {
+                if expect_include == (account == signers.first().expect("No signatures in transaction")) {
```

Comment:
The fee payer should always be the paymaster, I don't see any reason to make this configurable

Reply:
Good catch, `FeePayer` is actually a duplicate of the `Sponsor` variant. I'll remove

Reply:
We already have a check that ensures the paymaster account is the fee payer [here](https://github.com/fogo-foundation/fogo-sessions/blob/a96383b3eed0b562675b5668aa32b5004e9f4314/services/paymaster/src/api.rs#L140-L149)